CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Dead documentation link in README.md
 - `FullscreenToggleButton` being visible although `ViewMode.Fullscreen` is not available
 
+### Added
+- Support `TimelineMarkers` in live streams
+
```

Comment:
Missed that previously. Please move the `Added` section before the `Fixed` section.

Reply:
Addressed in https://github.com/bitmovin/bitmovin-player-ui/pull/310/commits/08b99dd7f70524693ae72bea92a180ff40fa4c7d